coursedata/pages/for-teachers/en.yaml

```diff
@@ -0,0 +1,91 @@
+title: "Hedy documentation"
+sections:
+    1:
+        title: "Introduction"
+        subsections:
+            1:
```

Comment:
Why give them numbers in the first place if you're never referring to them anywhere? Could this not just be a list?

I think you want the `section`s to have string keys (since they seem to represent different "pages", so you could potentially link them in the URL bar), but probably not the subsections (as they seem to be rendered onto a single page):

```
title: "Hedy documentation"
sections:
    introduction:
        title: "Introduction"
        subsections:
            - title: What's hedy
              text: <TODO>
            - title: Target audience
              text: <TODO>
```

Also, since you're ultimately going to render this to:

```
<h2>What's Hedy</h2>

(Text for What's hedy here)

<h2>Target audience</h2>

(Text here)
```

Instead of a list of sections, couldn't you store it as MarkDown inside the YAML:

```
## What's Hedy

(Text here)

## Target audience

(Text here)
```

What's the advantage of splitting every section into a separate Python object?

(Or is it just so that user's don't have control over the heading level?)

Reply:
Agree that the numbers don't make any sense! Will take a look to improve the structure (not that familiar with .yaml files (yet)).. The main reason for splitting is indeed to don't give the translators any control on the front-end level. I noticed that currently .md files have mixed styling due to mistakes being made on the translation side. Translating headers but forgetting or miscounting the amount of # characters needed to be consistent.